// @sapper/internal/manifest-client.mjs
// This file is generated by Sapper — do not edit it!
export { default as Root } from '../../../routes/_layout.svelte';
export { preload as root_preload } from './shared';
export { default as ErrorComponent } from '../../../routes/_error.svelte';

export const ignore = [/^\/photos\.json$/, /^\/photos\/([^\/]+?)\.json$/, /^\/til\.json$/, /^\/til\/([^\/]+?)\.json$/];

export const components = [
	{
		js: () => import("../../../routes/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:index.svelte__"
	},
	{
		js: () => import("../../../routes/servicios/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:servicios/index.svelte__"
	},
	{
		js: () => import("../../../routes/servicios/adidas-gmr.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:servicios/adidas-gmr.svelte__"
	},
	{
		js: () => import("../../../routes/servicios/annemasse.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:servicios/annemasse.svelte__"
	},
	{
		js: () => import("../../../routes/servicios/bicielx.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:servicios/bicielx.svelte__"
	},
	{
		js: () => import("../../../routes/servicios/cooltra.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:servicios/cooltra.svelte__"
	},
	{
		js: () => import("../../../routes/servicios/emt.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:servicios/emt.svelte__"
	},
	{
		js: () => import("../../../routes/services/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:services/index.svelte__"
	},
	{
		js: () => import("../../../routes/services/adidas-gmr.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:services/adidas-gmr.svelte__"
	},
	{
		js: () => import("../../../routes/services/annemasse.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:services/annemasse.svelte__"
	},
	{
		js: () => import("../../../routes/services/bicielx.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:services/bicielx.svelte__"
	},
	{
		js: () => import("../../../routes/services/cooltra.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:services/cooltra.svelte__"
	},
	{
		js: () => import("../../../routes/services/emt.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:services/emt.svelte__"
	},
	{
		js: () => import("../../../routes/photos/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:photos/index.svelte__"
	},
	{
		js: () => import("../../../routes/photos/[slug].svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:photos/[slug].svelte__"
	},
	{
		js: () => import("../../../routes/bio.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:bio.svelte__"
	},
	{
		js: () => import("../../../routes/til/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:til/index.svelte__"
	},
	{
		js: () => import("../../../routes/til/[slug].svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:til/[slug].svelte__"
	}
];

export const routes = (d => [
	{
		// index.svelte
		pattern: /^\/$/,
		parts: [
			{ i: 0 }
		]
	},

	{
		// servicios/index.svelte
		pattern: /^\/servicios\/?$/,
		parts: [
			{ i: 1 }
		]
	},

	{
		// servicios/adidas-gmr.svelte
		pattern: /^\/servicios\/adidas-gmr\/?$/,
		parts: [
			null,
			{ i: 2 }
		]
	},

	{
		// servicios/annemasse.svelte
		pattern: /^\/servicios\/annemasse\/?$/,
		parts: [
			null,
			{ i: 3 }
		]
	},

	{
		// servicios/bicielx.svelte
		pattern: /^\/servicios\/bicielx\/?$/,
		parts: [
			null,
			{ i: 4 }
		]
	},

	{
		// servicios/cooltra.svelte
		pattern: /^\/servicios\/cooltra\/?$/,
		parts: [
			null,
			{ i: 5 }
		]
	},

	{
		// servicios/emt.svelte
		pattern: /^\/servicios\/emt\/?$/,
		parts: [
			null,
			{ i: 6 }
		]
	},

	{
		// services/index.svelte
		pattern: /^\/services\/?$/,
		parts: [
			{ i: 7 }
		]
	},

	{
		// services/adidas-gmr.svelte
		pattern: /^\/services\/adidas-gmr\/?$/,
		parts: [
			null,
			{ i: 8 }
		]
	},

	{
		// services/annemasse.svelte
		pattern: /^\/services\/annemasse\/?$/,
		parts: [
			null,
			{ i: 9 }
		]
	},

	{
		// services/bicielx.svelte
		pattern: /^\/services\/bicielx\/?$/,
		parts: [
			null,
			{ i: 10 }
		]
	},

	{
		// services/cooltra.svelte
		pattern: /^\/services\/cooltra\/?$/,
		parts: [
			null,
			{ i: 11 }
		]
	},

	{
		// services/emt.svelte
		pattern: /^\/services\/emt\/?$/,
		parts: [
			null,
			{ i: 12 }
		]
	},

	{
		// photos/index.svelte
		pattern: /^\/photos\/?$/,
		parts: [
			{ i: 13 }
		]
	},

	{
		// photos/[slug].svelte
		pattern: /^\/photos\/([^\/]+?)\/?$/,
		parts: [
			null,
			{ i: 14, params: match => ({ slug: d(match[1]) }) }
		]
	},

	{
		// bio.svelte
		pattern: /^\/bio\/?$/,
		parts: [
			{ i: 15 }
		]
	},

	{
		// til/index.svelte
		pattern: /^\/til\/?$/,
		parts: [
			{ i: 16 }
		]
	},

	{
		// til/[slug].svelte
		pattern: /^\/til\/([^\/]+?)\/?$/,
		parts: [
			null,
			{ i: 17, params: match => ({ slug: d(match[1]) }) }
		]
	}
])(decodeURIComponent);
