// @sapper/internal/manifest-server.mjs
// This file is generated by Sapper — do not edit it!
import * as route_0 from "../../../routes/photos/index.json.js";
import * as route_1 from "../../../routes/photos/[slug].json.js";
import * as route_2 from "../../../routes/til/index.json.js";
import * as route_3 from "../../../routes/til/[slug].json.js";
import component_0 from "../../../routes/index.svelte";
import component_1 from "../../../routes/servicios/index.svelte";
import component_2 from "../../../routes/servicios/adidas-gmr.svelte";
import component_3 from "../../../routes/servicios/annemasse.svelte";
import component_4 from "../../../routes/servicios/bicielx.svelte";
import component_5 from "../../../routes/servicios/cooltra.svelte";
import component_6 from "../../../routes/servicios/emt.svelte";
import component_7 from "../../../routes/services/index.svelte";
import component_8 from "../../../routes/services/adidas-gmr.svelte";
import component_9 from "../../../routes/services/annemasse.svelte";
import component_10 from "../../../routes/services/bicielx.svelte";
import component_11 from "../../../routes/services/cooltra.svelte";
import component_12 from "../../../routes/services/emt.svelte";
import component_13, { preload as preload_13 } from "../../../routes/photos/index.svelte";
import component_14, { preload as preload_14 } from "../../../routes/photos/[slug].svelte";
import component_15 from "../../../routes/bio.svelte";
import component_16, { preload as preload_16 } from "../../../routes/til/index.svelte";
import component_17, { preload as preload_17 } from "../../../routes/til/[slug].svelte";
import root from "../../../routes/_layout.svelte";
import error from "../../../routes/_error.svelte";

const d = decodeURIComponent;

export const manifest = {
	server_routes: [
		{
			// photos/index.json.js
			pattern: /^\/photos\.json$/,
			handlers: route_0,
			params: () => ({})
		},

		{
			// photos/[slug].json.js
			pattern: /^\/photos\/([^\/]+?)\.json$/,
			handlers: route_1,
			params: match => ({ slug: d(match[1]) })
		},

		{
			// til/index.json.js
			pattern: /^\/til\.json$/,
			handlers: route_2,
			params: () => ({})
		},

		{
			// til/[slug].json.js
			pattern: /^\/til\/([^\/]+?)\.json$/,
			handlers: route_3,
			params: match => ({ slug: d(match[1]) })
		}
	],

	pages: [
		{
			// index.svelte
			pattern: /^\/$/,
			parts: [
				{ name: "index", file: "index.svelte", component: component_0 }
			]
		},

		{
			// servicios/index.svelte
			pattern: /^\/servicios\/?$/,
			parts: [
				{ name: "servicios", file: "servicios/index.svelte", component: component_1 }
			]
		},

		{
			// servicios/adidas-gmr.svelte
			pattern: /^\/servicios\/adidas-gmr\/?$/,
			parts: [
				null,
				{ name: "servicios_adidas$45gmr", file: "servicios/adidas-gmr.svelte", component: component_2 }
			]
		},

		{
			// servicios/annemasse.svelte
			pattern: /^\/servicios\/annemasse\/?$/,
			parts: [
				null,
				{ name: "servicios_annemasse", file: "servicios/annemasse.svelte", component: component_3 }
			]
		},

		{
			// servicios/bicielx.svelte
			pattern: /^\/servicios\/bicielx\/?$/,
			parts: [
				null,
				{ name: "servicios_bicielx", file: "servicios/bicielx.svelte", component: component_4 }
			]
		},

		{
			// servicios/cooltra.svelte
			pattern: /^\/servicios\/cooltra\/?$/,
			parts: [
				null,
				{ name: "servicios_cooltra", file: "servicios/cooltra.svelte", component: component_5 }
			]
		},

		{
			// servicios/emt.svelte
			pattern: /^\/servicios\/emt\/?$/,
			parts: [
				null,
				{ name: "servicios_emt", file: "servicios/emt.svelte", component: component_6 }
			]
		},

		{
			// services/index.svelte
			pattern: /^\/services\/?$/,
			parts: [
				{ name: "services", file: "services/index.svelte", component: component_7 }
			]
		},

		{
			// services/adidas-gmr.svelte
			pattern: /^\/services\/adidas-gmr\/?$/,
			parts: [
				null,
				{ name: "services_adidas$45gmr", file: "services/adidas-gmr.svelte", component: component_8 }
			]
		},

		{
			// services/annemasse.svelte
			pattern: /^\/services\/annemasse\/?$/,
			parts: [
				null,
				{ name: "services_annemasse", file: "services/annemasse.svelte", component: component_9 }
			]
		},

		{
			// services/bicielx.svelte
			pattern: /^\/services\/bicielx\/?$/,
			parts: [
				null,
				{ name: "services_bicielx", file: "services/bicielx.svelte", component: component_10 }
			]
		},

		{
			// services/cooltra.svelte
			pattern: /^\/services\/cooltra\/?$/,
			parts: [
				null,
				{ name: "services_cooltra", file: "services/cooltra.svelte", component: component_11 }
			]
		},

		{
			// services/emt.svelte
			pattern: /^\/services\/emt\/?$/,
			parts: [
				null,
				{ name: "services_emt", file: "services/emt.svelte", component: component_12 }
			]
		},

		{
			// photos/index.svelte
			pattern: /^\/photos\/?$/,
			parts: [
				{ name: "photos", file: "photos/index.svelte", component: component_13, preload: preload_13 }
			]
		},

		{
			// photos/[slug].svelte
			pattern: /^\/photos\/([^\/]+?)\/?$/,
			parts: [
				null,
				{ name: "photos_$slug", file: "photos/[slug].svelte", component: component_14, preload: preload_14, params: match => ({ slug: d(match[1]) }) }
			]
		},

		{
			// bio.svelte
			pattern: /^\/bio\/?$/,
			parts: [
				{ name: "bio", file: "bio.svelte", component: component_15 }
			]
		},

		{
			// til/index.svelte
			pattern: /^\/til\/?$/,
			parts: [
				{ name: "til", file: "til/index.svelte", component: component_16, preload: preload_16 }
			]
		},

		{
			// til/[slug].svelte
			pattern: /^\/til\/([^\/]+?)\/?$/,
			parts: [
				null,
				{ name: "til_$slug", file: "til/[slug].svelte", component: component_17, preload: preload_17, params: match => ({ slug: d(match[1]) }) }
			]
		}
	],

	root,
	root_preload: () => {},
	error
};

export const build_dir = "__sapper__/build";

export const src_dir = "src";

export const dev = false;
